app/scripts/directives/podMetrics.js

```diff
@@ -130,6 +146,18 @@ angular.module('openshiftConsole')
         // Show last hour by default.
         scope.options.timeRange = _.head(scope.options.rangeOptions);
 
+        function generateVolumeDataSets() {
+          angular.forEach(scope.pod.spec.volumes, function(volumeData) {
+            if (volumeData.persistentVolumeClaim !== undefined) {
+              volumeDataSets.push({
+                id: "filesystem/usage",
```

Comment:
According to the "filesystem/usage" metric definitions, we can only get the pod filesystem usage. Maybe need to add the "pvc/usage" metric definitions.  I am following the cadvisor for this. Thanks.